HPXMLtoOpenStudio/resources/EPvalidator.stron

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where did the .stron file extension come from? Seems like it's typically .sch or .xml.